### DISCUSSION TITLE
ModuleNotFoundError: No module named 'cv2'

### DISCUSSION THREAD

**Discussion by Dentonknifeworks** [Curtis2012/clearskyalarm#8] [Q&A] [answered]
Ok was kinda surprised this is missing.
So I tried  
sudo pip3 install opencv-contrib-python
This took about 2 hrs and then errored out.
Looking for a suggestion for getting the Module cv2 installed.
My python is version 3.7
Thanks

**Comment by Curtis2012** [comment, marked as answer]
I initially had some issues with installing openCV also. There are many variations on the install process posted on the internet, some are unnecessarily complicated. In the end, I installed two openCV packages using PIP.  The instructions at the link below worked for me, with one cavaet: it appears that the statement in the instructions that the main modules are included in the contrib package is incorrect.  I first installed the main modules package (opencv-python) and then the contrib package. This worked for me. 

OpenCV is large, but why it would take 2 hours to install and then fail, I do not know. The pip installs always completed successfully in 
my case. 


https://pypi.org/project/opencv-python/

**Reply by Dentonknifeworks** [reply]
Thanks, I will try it. But I don't see the link.

**Reply by Curtis2012** [reply]
Oops. Here it is again. Also edited reply to include.

https://pypi.org/project/opencv-python/
